src/content/tools/pub/cmd/pub-lish.md

```diff
@@ -36,5 +36,14 @@ In the event of warnings, your package *is* uploaded.
 To ensure that your package has no warnings before uploading,
 either don't use `--force`, or use `--dry-run` first.
 
+### `--skip-validation`
+
+Publishes without going through the validation process or resolving dependencies.
+This is useful for advanced users who knows why the validation might fail and wishes to side step a particularly issues.
```

Comment:
Perhaps clarify that this is only the client side validations

Reply:
Added `client-side` to the first line :D